server/tests/ballot_comparison/snapshots/snap_test_ballot_comparison.py

```diff
@@ -55,7 +55,7 @@
 J1,0002,BATCH2,5,0002000175,"Round 1: 0.128937575131137250, 0.240487859312182291",AUDITED,Choice 1-2,Choice 1-2,,,Choice 2-2,Choice 2-2,,\r
 J2,0001,BATCH1,1,0001013415,Round 1: 0.228946820159681463,AUDITED,"Choice 1-1, INVALID_WRITE_IN",Choice 1-1,,,"Choice 2-1, INVALID_WRITE_IN",Choice 2-1,,\r
 J2,0001,BATCH1,3,0001013417,Round 1: 0.457121710197159606,AUDITED,Choice 1-1,Choice 1-1,,,Choice 2-1,Choice 2-1,,\r
-J2,0001,BATCH2,3,0001000417,Round 1: 0.269793733438455805,AUDITED,"Choice 1-1, Choice 1-2",Overvote,Choice 1-1: -1; Choice 1-2: -1,,"Choice 2-1, Choice 2-3",Overvote,Choice 2-1: -1; Choice 2-3: -1,\r
+J2,0001,BATCH2,3,0001000417,Round 1: 0.269793733438455805,AUDITED,"Choice 1-1, Choice 1-2",Overvote,Choice 1-2: -1,,"Choice 2-1, Choice 2-3",Overvote,,\r
```

Comment:
I'm looking at this snapshot diff more closely and noticing something off.

The following

```
Audit result: Choice 1-1, Choice 1-2
CVR result:   Overvote
```

is still resulting in a discrepancy of `Choice 1-2: -1` (previously `Choice 1-1: -1; Choice 1-2: -1`).

In contrast, the following

```
Audit result: Choice 2-1, Choice 2-3
CVR result:   Overvote
```

is expectedly resulting in no discrepancy (prevoiusly `Choice 2-1: -1; Choice 2-3: -1`).

I have a rough idea here what's going on and am prepping an edit to account for this. Because I can't push to this branch from a fork, I'll open a new PR.

Reply:
As far as I can tell, when a contest is overvoted, we do save an "o" for every candidate: https://github.com/votingworks/arlo/blob/main/server/api/cvrs.py#L885-L900